src/pages/Login/Login.test.tsx

```diff
@@ -106,6 +107,8 @@ describe("Get SMS PIN button", () => {
   });
 
   describe("on click", () => {
+    afterEach(() => localStorage.removeItem(OTP_AUTH_KEY));
```

Comment:
I'm a bit conflicted on this.
Under the `on click` test suite, there are no files that actually care about localStorage. But I put this code because I though we ought to clean up after any effects that are introduced in a test to ensure test isolation.
Thus it matters not if localStorage is used, its more like we want to clean up after whatever a test might introduce.
Otherwise we might end up with bugs that are hard to trace due to effects "leaking" into other tests.

Reply:
Good call. There might be some chances, ( and it can get frustrating sometimes 😆 )

Since these tests are mocking and not setting any localstorage, a ` beforeAll` should be fine for removing localStorage

Reply:
Yeah, an `afterAll` is also alright as a "just in case"

Reply:
@bharat619 some slight miscommunication from my side. Actually localStorage is set within this test suite. What I meant by don't care was in the sense that I don't make any assertions on its value within this suite.
Thus, I'll have to continue using `afterEach`.